### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by tuexss**
I tried to add a bit of consistency and remove unused or unnecessary lines of code. I hope this helps a bit.

### CODE REVIEW

File: lib/github_changelog_generator.rb [hunk 1]
```diff
@@ -139,7 +139,7 @@ def fetch_merged_at_pull_requests
       }
 
       if @options[:verbose]
-        puts 'Fetching merged dates... Done!'
+        puts "Fetching merged dates: Done!"
```
Comment (skywinder) [inline lib/github_changelog_generator.rb]: Thanks for cleanup! I think there is should be single quotes, since you don't use any parameters.

Comment (tuexss) [inline lib/github_changelog_generator.rb]: I would've gone with http://viget.com/extend/just-use-double-quoted-ruby-strings and always use double quotes. But if you prefer mixed style, I can revert this part :)

**Comment by skywinder** [comment]
Wow! @tuexss it seems you are deeply inspect this code and fix bunch of different typos and logic issues! Now it looks much healthy! :gem: 
Thanks a lot! :+1:

File: lib/github_changelog_generator.rb (hunk 1 above)
Comment (skywinder) [inline lib/github_changelog_generator.rb]: @tuexss it really makes sense. I just new in ruby, and consider, if RubyMine and rubocop pronouncedly highlight this - it should be like that "a priori". So, I thinking to move to double quote and revert this "fancy" PR :)

.. and add to rubocom.yml.

    Style/StringLiterals:
      EnforcedStyle: double_quotes

Thanks, for point it out!

**Comment by tuexss** [comment]
Yes, i'm also quite new in ruby. Thanks for the merge!

**Pull request merged**